sui/operators.js

```diff
@@ -0,0 +1,301 @@
+const { Command, Option } = require('commander');
+const { Transaction } = require('@mysten/sui/transactions');
+const { bcs } = require('@mysten/sui/bcs');
+const { printInfo, loadConfig } = require('../common/utils');
+const { operatorsStruct } = require('./types-utils');
+const { addBaseOptions, addOptionsToCommands, parseSuiUnitAmount } = require('./cli-utils');
+const { getWallet, printWalletInfo, broadcast } = require('./sign-utils');
+const { getBcsBytesByObjectId, findOwnedObjectId } = require('./utils');
+
+async function getGasCollectorCapId(client, gasServiceConfig, operatorsConfig) {
```

Comment:
Do we need this? We store the gas collector cap id when deploying the gas service. Just read that? We could add a separate cmd in this script that checks if the gas collector (or user provided) cap id is actually stored in the Operators though